src/cloud-api-adaptor/docker/README.md

```diff
@@ -80,7 +80,7 @@ For changing the CAA image to your custom built image (eg. `quay.io/myuser/cloud
 you can use the following:
 
 ```bash
-kubectl set image ds/cloud-api-adaptor-daemonset -n confidential-containers-system cloud-api-adaptor-con=quay.io/myuser/cloud-api-adaptor
+kubectl set image ds/peerpodconfig-ctrl-caa-daemon -n confidential-containers-system caa-pod=quay.io/myuser/cloud-api-adaptor
```

Comment:
nit: we could do all the renaming in a single commit.

Reply:
Done